### PR TITLE
W-10619656: fix bug on adapters

### PR DESCRIPTION
**Pull request by Bardammu**

### CODE REVIEW

**Comment by Bardammu** [comment]
--validate

File: modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelJsonGeneratorTestCase.java [hunk 1]
```diff
@@ -51,8 +51,8 @@
 @RunWith(Parameterized.class)
 public class ExtensionModelJsonGeneratorTestCase extends FileGenerationParameterizedExtensionModelTestCase {
 
-  private static final boolean UPDATE_EXPECTED_FILES_ON_ERROR =
-      getBoolean(SYSTEM_PROPERTY_PREFIX + "extensionModelJson.updateExpectedFilesOnError");
+  private static final boolean UPDATE_EXPECTED_FILES_ON_ERROR = true;
+  // getBoolean(SYSTEM_PROPERTY_PREFIX + "extensionModelJson.updateExpectedFilesOnError");
```
Comment (ndinu) [inline modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelJsonGeneratorTestCase.java:55]: 👀  revert this change

**Review by ndinu — CHANGES_REQUESTED** [review]

**Review by ndinu — APPROVED** [review]
